README.md

```
@@ -320,7 +320,7 @@ If you use Codex, this repository ships equivalent workflow skills in
 | `implement` | Detecta a camada, implementa e testa |
 | `ship-issue` | Commit e PR com descrição baseada no diff e todos os metadados da issue |
 | `handle-pr-findings` | Corrige findings, atualiza o PR e responde/resolve threads autonomamente |
-| `review-pr <PR|ref>` | Revisão cruzada sob demanda de um PR ou referência explícita, por camada |
+| `review-pr <PR|ref>` | Revisão cruzada sob demanda ou re-review incremental de um PR/ref, por camada |
```

markdownlint-cli2 (0.23.2)

[warning] Table column count
Expected: 2; Actual: 3; Too many cells, extra data will be missing

(MD056, table-column-count)